prow/tide/tide.go

```diff
@@ -714,19 +771,26 @@ func (m *mergeChecker) repoMethods(orgRepo config.OrgRepo) (map[types.PullReques
 // isAllowed checks if a PR does not have merge conflicts and requests an
 // allowed merge method. If there is no error it returns a string explanation if
 // not allowed or "" if allowed.
-func (m *mergeChecker) isAllowed(pr *PullRequest) (string, error) {
+func (m *mergeChecker) isAllowed(crc *CodeReviewCommon) (string, error) {
```

Comment:
If the `mergeChecker` is GitHub specific then this should just take PullRequest. Or is the plan to make this an interface?

Reply:
Either way, `prMergeMethod` only makes sense for GitHub PRs so that should just take a PullRequest.